cmd/channel/main.go

```diff
@@ -33,7 +33,7 @@ const (
 // Variables
 var (
 	logger        *zap.Logger
-	masterURL     = flag.String("masterurl", "", "The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
+	serverURL     = flag.String("server", "", "The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
```

Comment:
can we potentially rework to just use the pkg helper, if there are no other flags anyway?

Reply:
I don't have much context on this package or why it doesn't use pkg tbh, but that sure sounds good if we could: @travis-minke-sap @eric-sap @slinkydeveloper wdyt?

Reply:
it looks to me like it's not an entirely trivial swap we pass `masterURL` to `InitializeKafkaChannelLister` below, so my vote would be to merge this to fix the immediate issue and then look at adopting helpers from pkg separately.. but defer to folks with more context on all these things :).

Reply:
indeed! SG!

Reply:
The changes look fine and I agree on the approach (deferring rework to pkg helper).  The only reason for not using some "pkg helper" is just ignorance of its existence.  Can someone point to a github location of this code and I'll take a look at incorporating it into a future delivery?

Reply:
Sure thing: https://github.com/knative/pkg/blob/master/injection/sharedmain/main.go. There are a few `OrDie` helpers, that generally help setting up a component (getting a config etc.)

Reply:
👍

Reply:
/approve

Reply:
/lgtm